File: src/main/java/com/rs/java/game/area/multi/KingBlackDragonMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.area.Area;

public final class KingBlackDragonMulti extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "King Black Dragon Multi";
	}

	public KingBlackDragonMulti() {
		super(9033);
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}

}

File: src/raynna/game/world/area/areas/multi/KingBlackDragonMulti.kt
package raynna.game.world.area.areas.multi

import raynna.game.world.area.Area

class KingBlackDragonMulti : Area(9033) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "King Black Dragon Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}
